### PR TITLE
Add manual tests for web manifest

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: appmanifest/theme_color-member/theme_color-member-red.webmanifest.headers [hunk 1]
```diff
@@ -0,0 +1 @@
+Content-Type: application/manifest+json; charset=UTF-8
```
Comment (marcoscaceres) [inline appmanifest/theme_color-member/theme_color-member-red.webmanifest.headers:1]:
```suggestion
Content-Type: application/manifest+json; charset=utf-8
```

**Review by anssiko — APPROVED** [review]

File: appmanifest/META.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 spec: https://w3c.github.io/manifest/
 suggested_reviewers:
-  - anssiko
   - kenchris
   - marcoscaceres
   - mgiuca
-  - mounirlamouri
```
Comment (mounirlamouri) [inline appmanifest/META.yml:7]: This is outrageous! 😄

Comment (marcoscaceres) [reply appmanifest/META.yml:7]: heh, you really want to review tests?

Comment (mounirlamouri) [reply appmanifest/META.yml:7]: Sorry, I thought the 😄 would be a clue that it was a tongue in cheek reply 🙈

Comment (marcoscaceres) [reply appmanifest/META.yml:7]: just checking :) will spare you from this.

**Pull request merged**